.circleci/config.yml

```diff
@@ -33,6 +33,14 @@ jobs:
             sudo apt-get update -qq && sudo apt-get install -y build-essential postgresql-client
             echo 'export PATH=/usr/lib/postgresql/9.6/bin/:$PATH' >> $BASH_ENV
 
+      - run:
+          name: Install flyway
+          command: |
+            sudo apt-get update && sudo apt-get install -y default-jdk
+            mkdir flyway
+            curl -s https://repo.maven.apache.org/maven2/org/flywaydb/flyway-commandline/4.2.0/flyway-commandline-4.2.0-linux-x64.tar.gz | tar zxv -C flyway
```

Comment:
How often does `flyway` release updates?  It looks like roughly once a month for patch level updates, and every few months for minor and major releases.  I see that [version 5.0.0](https://flywaydb.org/documentation/releaseNotes) is in the works, which appears to include some updates around the minimum supported version of Java and Maven, JDBC driver updates, etc., all of which are probably important to stay on top of given security updates, vulnerability patches, etc.

I also ask because I'm wondering how we can best account for updates to this tool and make sure it's relatively easy/straightforward to maintain in our build pipeline.

Reply:
Major releases once or twice a year appears to be the frequency. I agree that we should keep on top of updates and security patches. As FEC starts to use `flyway` for more of their projects, it will be more important to do this. 

As we use `flyway` in its sweet spot (we don't rely on any edge case behavior), we should be able to easily upgrade to patch releases as well.